[RU] Update `Guides/Audio_editing` (#15032)
ppy/osu-wiki: wiki/Guides/Audio_editing/ru.md

# Руководство по редактированию аудио

[osu!academy](/wiki/Community/Video_series/osu!academy) рассказали об этом в [Episode 15: Audio Encoding (4:02)](https://www.youtube.com/watch?v=muu3HkG38kk). В этом выпуске также рассказывается, как установить и использовать Audacity с возможностью экспорта в `.mp3` (LAME).

> Это руководство покажет, как внести незначительные изменения в аудиофайлы и подготовить их к маппингу. Дополнив его, вы можете помочь остальным мапперам научиться обрабатывать аудиофайлы для любых нужд.

Reviewer: мне кажется, так чуть получше (второе предложение, мне кажется, вообще лишнее, но это уже к оригиналу вопросы)
```suggestion
Это руководство покажет, как внести незначительные изменения в аудиофайлы и подготовить их к маппингу. Дополнив его, вы поможете остальным мапперам научиться обрабатывать аудио для разных целей.
```

*При обработке аудио вы не ограничены перечисленными здесь программами — это выбор авторов руководства. Если вы знаете другие подобные инструменты и в состоянии объяснить, как ими пользоваться, можете также их добавить.*

## Audacity

::: alert-notice

> **Внимание**

Reviewer: новую версию ASC надо тут поправить (но это будет потом), но это не заслуживает **внимания**, скорее, это просто **примечание**
```suggestion
**Примечание**
```

> В данной статье идёт речь об Audacity 3.4.2

Reviewer 2: ```suggestion
На момент написания статьи использовалась версия Audacity 3.4.2
```

:::

> [Audacity](https://www.audacityteam.org/download) — это бесплатное программное обеспечение для редактирования и записи аудио с открытым исходным кодом. Для экспорта аудиофайлов в формате MP3 оно использует библиотеку кодирования [LAME](https://lame.sourceforge.io). Данная библиотека встроена в Audacity на Windows и macOS.

Reviewer: раз уж ты за это предложение схватился
```suggestion
[Audacity](https://www.audacityteam.org/download) — это бесплатное программное обеспечение с открытым исходным кодом для редактирования и записи аудио. Для экспорта аудиофайлов в формате MP3 оно использует библиотеку кодирования [LAME](https://lame.sourceforge.io). Данная библиотека встроена в Audacity на Windows и macOS.
```

> Пользователям на Linux следует обратиться к [справочному руководству Audacity](https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html#linlame) для подробной информации, так как некоторые дистрибутивы Linux при установке Audacity могут не включать в себя LAME, однако большинство делают это.

Reviewer: ```suggestion
Пользователям Linux следует обратиться за подробностями к [справочному руководству Audacity](https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html#linlame), так как некоторые дистрибутивы Linux при установке Audacity могут не устанавливать LAME (но большинство всё-таки устанавливает).
```

### Понижение битрейта

::: alert-note

> **Примечание:** Подробнее о сжатии файлов [Сжатие файлов](/wiki/Guides/Compressing_files)

Reviewer: ```suggestion
**Примечание:** Подробнее о сжатии файлов см. [Сжатие файлов](/wiki/Guides/Compressing_files)
```

:::

Установите и откройте Audacity, затем выполните следующие действия:

1. Откройте файл `.mp3`, битрейт которого нужно понизить.

> 2. Нажмите `Ctrl` + `Shift` + `E`, или выберите пункт меню `Файл` -> `Экспорт аудио...`.

Reviewer: с одним «или» запятая тут не нужна
по повоту «в меню» — это мелочь, можешь как применять, так и оставить (но я бы, само собой, применил) 
```suggestion
2. Нажмите `Ctrl` + `Shift` + `E` или выберите в меню `Файл` -> `Экспорт аудио...`.
```

3. В `Формат:` выберите `Файлы MP3`.
4. В `Параметры аудио` поменяйте следующие настройки:
   1. `Режим битрейта`: `Пресет`
   2. `Качество`: `Средний, 145-185 кбит/с`

> 5. Отредактируйте `Имя файла:` и `Папка:` чтобы определить место и название, под которым вы хотите сохранить файл.

Reviewer: ```suggestion
5. Отредактируйте `Имя файла:` и `Папка:` чтобы указать место и название, под которым вы хотите сохранить файл.
```

>    - Вы также можете использовать кнопку `Обзор...`.

Reviewer: ```suggestion
   - Указать место также можно с помощью кнопки `Обзор...`.
```

6. Если вы хотите отредактировать метаданные файла, нажмите `Правка метаданных...`.
7. Нажмите `ОК`, когда закончите редактировать метаданные.
8. Нажмите `Экспорт` чтобы экспортировать аудиофайл.

### Зацикливание

Установите и откройте Audacity, затем выполните следующие действия:

1. Откройте файл `.mp3`, который необходимо зациклить.
2. Зажав и перетащив курсор мыши, выделите участки, которые вы хотите зациклить.

>    - Изредка это весь трек целиком.

Reviewer: ```suggestion
   - В некоторых случаях это может быть весь трек целиком.
```

   - Иногда для зацикливания лучше всего подходит припев или его часть.

> 3. Нажмите `Ctrl` + `C` или выберите пункт меню `Правка` -> `Копировать`.

Reviewer: ```suggestion
3. Нажмите `Ctrl` + `C` или выберите в меню `Правка` -> `Копировать`.
```

4. Найдите место, куда скопированная часть лучше всего подойдёт.

>    - Это может быть конец выделенного участка.

Reviewer: может быть, а может и не быть? в русском как-то более определённо с этим надо поступать, автор же знает, чего он хочет
```suggestion
   - Например, сразу после выделенного участка.
```

> 5. Нажмите `Ctrl` + `V` или выберите пункт меню `Правка` -> `Вставить`.

Reviewer: ```suggestion
5. Нажмите `Ctrl` + `V` или выберите в меню `Правка` -> `Вставить`.
```

6. Прослушайте аудио целиком и убедитесь, что луп звучит хорошо.
7. Повторите эти шаги по мере необходимости.
8. [Экспортируйте](#понижение-битрейта) аудиофайл.

### Обрезка

Установите и откройте Audacity, затем выполните следующие действия:

1. Откройте файл `.mp3`, который необходимо обрезать.

> 2. Зажав и перетащив курсор мыши, выделите участки, которые вы хотите зациклить.

Reviewer: ```suggestion
2. Зажав и перетащив курсор мыши, выделите участки, которые вы хотите вырезать.
```

   - Чаще всего это конец песни, который не хочется мапать.

> 3. Нажмите `Delete`, `Ctrl` + `K` или выберите пункт меню `Правка` -> `Удалить`.

Reviewer: а вот тут запятая уже нужна..
```suggestion
3. Нажмите `Delete`, `Ctrl` + `K`, или выберите в меню `Правка` -> `Удалить`.
```

4. Выделите последние несколько секунд.

> 5. Выберите пункт меню `Эффекты` -> `Фейдинг` -> `Фейд-спад`.

Reviewer: ```suggestion
5. Выберите в меню `Эффекты` -> `Фейдинг` -> `Фейд-спад`.
```

6. [Экспортируйте](#понижение-битрейта) аудиофайл.

## mp3DirectCut

::: alert-notice

> **Внимание**

Reviewer: ```suggestion
**Примечание**
```

> В данной статье идёт речь об mp3DirectCut 2.36.

Reviewer 2: ```suggestion
На момент написания статьи использовалась версия mp3DirectCut 2.36.
```

:::

> [mp3DirectCut](https://mpesch3.de) — бесплатная программа для редактирования аудио. Этот инструмент рекомендуется только для изменения громкости или обрезки аудио, поскольку он не перекодирует аудиофайл, а значит, и потеря качества будет минимальна.

Reviewer: в твоём варианте это так звучит, как будто больше ни для чего её нельзя порекомендовать, смысл в оригинале — её стоит использовать, если требуются только эти изменения
```suggestion
[mp3DirectCut](https://mpesch3.de) — бесплатная программа для редактирования аудио. Рекомендуется к использованию, когда нужно только изменить громкость или обрезать аудио, поскольку она не перекодирует аудиофайл, а значит, и потеря качества будет минимальна.
```

### Понижение битрейта

::: alert-notice
**Внимание**
Для использования этой функции требуется 32-разрядный файл библиотеки.
:::

1. Откройте файл `.mp3`, битрейт которого вы хотите понизить.

> 2. Dыберите пункт меню `Установки` -> `Конфигурация...` или нажмите `F11`.

Reviewer: jсторожно
```suggestion
2. Выберите в меню `Установки` -> `Конфигурация...` или нажмите `F11`.
```

Reviewer: (а settings правда там переведено как `установки`?..)

Author: Правда, также прошу прощения за опечатку.

> 3. Перейдите во вкладку `Кодировщик`.

Reviewer: только так
```suggestion
3. Перейдите на вкладку `Кодировщик`.
```

> 4. Включите одну из доступных библиотек кодирования.

Reviewer: если там нужно сделать выбор (или-или), на что намекает формулировка, то лучше так и написать: `Выберите`

```suggestion
4. Выберите одну из доступных библиотек кодирования.
```

   - `System`: библиотека кодирования ACM

>    - `Lame`: Файл `Lame_enc.dll` в установочной директории

Reviewer: мне кажется (но могу ошибаться, давно не накликивал ничего в винде), что «установочная директория» — это только место, куда программа *будет* установлена, а когда она уже там, обычно пишут (т.е.  про фактическое местоположение) «директория установки», или нет?
```suggestion
   - `Lame`: Файл `Lame_enc.dll` в папке с программой
```

> 5. Установите минимальный битрейт `128 kbit/s`, а максимальный `192 kbit/s`.

Reviewer: ```suggestion
5. Установите минимальный битрейт `128 kbit/s`, а максимальный — `192 kbit/s`.
```

6. Нажмите `Закрыть`.

> 7. Нажмите `Ctrl` + `W` или выберите пункт меню `Файл` -> `Сохранить всё аудио...`.

Reviewer: ```suggestion
7. Нажмите `Ctrl` + `W` или выберите в меню `Файл` -> `Сохранить всё аудио...`.
```

8. Откройте папку, куда вы хотите сохранить файл.
   - Там же можно переименовать файл.
9. Нажмите на кнопку `Сохранить`.

### Зацикливание

Установите и откройте mp3DirectCut, затем выполните следующие действия:

1. Откройте файл `.mp3`, который вы хотите зациклить.
2. Зажав и перетащив курсор мыши, выделите участки, которые вы хотите зациклить.

>    - Изредка это весь трек целиком.

Reviewer: ```suggestion
   - В некоторых случаях это может быть весь трек целиком.
```

   - Иногда для зацикливания лучше всего подходит припев или его часть.
3. Нажмите `Ctrl` + `C` или выберите пункт меню `Правка` -> `Копировать`.
4. Найдите место, куда скопированная часть лучше всего подойдёт.

>    - Это может быть конец выделенного участка.

Reviewer: ```suggestion
   - Например, сразу после выделенного участка.
```

> 5. Нажмите `Ctrl` + `V`или выберите пункт меню `Правка` -> `Вставить`.

Reviewer: ```suggestion
5. Нажмите `Ctrl` + `V`или выберите в меню `Правка` -> `Вставить`.
```

Author: Тут пробел пропущен, надо исправить)

> 5. Нажмите `Ctrl` + `V`или выберите пункт меню `Правка` -> `Вставить`.

Author: ```suggestion
5. Нажмите `Ctrl` + `V` или выберите в меню `Правка` -> `Вставить`.
```

6. Прослушайте аудио целиком и убедитесь, что луп звучит хорошо.
7. Повторите эти шаги по мере необходимости.

> 8. Нажмите `Ctrl` + `W`или выберите пункт меню `Файл` -> `Сохранить всё аудио...`:

Reviewer: ```suggestion
8. Нажмите `Ctrl` + `W`или выберите в меню `Файл` -> `Сохранить всё аудио...`:
```

Author: Тоже пробел

> 8. Нажмите `Ctrl` + `W`или выберите пункт меню `Файл` -> `Сохранить всё аудио...`:

Author: ```suggestion
8. Нажмите `Ctrl` + `W` или выберите в меню `Файл` -> `Сохранить всё аудио...`:
```

9. Откройте папку, куда вы хотите сохранить файл.
   - Там же можно переименовать файл.
10. Нажмите на кнопку `Сохранить`.

### Обрезка

Установите и откройте mp3DirectCut, затем выполните следующие действия:

1. Откройте файл `.mp3`, который необходимо обрезать.

> 2. Зажав и перетащив курсор мыши, выделите участки, которые вы хотите зациклить.

Reviewer: ```suggestion
2. Зажав и перетащив курсор мыши, выделите участки, которые вы хотите обрезать.
```

   - Чаще всего это конец песни, который не хочется мапать.
3. Нажмите `Delete`.
4. Выделите последние несколько секунд.
5. Нажмите `Ctrl` + `F` или выберите пункт меню `Правка` -> `Простое затухание/нарастание`.

> 6. Нажмите `Ctrl` + `W`или выберите пункт меню `Файл` -> `Сохранить всё аудио...`.

Reviewer: ```suggestion
5. Нажмите `Ctrl` + `F` или выберите в меню `Правка` -> `Простое затухание/нарастание`.
6. Нажмите `Ctrl` + `W`или выберите в меню `Файл` -> `Сохранить всё аудио...`.
```

Author: Опять пробел

> 6. Нажмите `Ctrl` + `W`или выберите пункт меню `Файл` -> `Сохранить всё аудио...`.

Author: ```suggestion
6. Нажмите `Ctrl` + `W` или выберите в меню `Файл` -> `Сохранить всё аудио...`.
```

7. Откройте папку, куда вы хотите сохранить файл.
   - Там же можно переименовать файл.
8. Нажмите на кнопку `Сохранить`.

### Регулировка громкости

Установите и откройте mp3DirectCut, затем выполните следующие действия:

1. Откройте файл `.mp3`, громкость которого вы хотите отрегулировать.
2. Нажмите `Ctrl` + `A`, чтобы выделить всё аудио.

> 3. Нажмите `Ctrl` + `G` или выберите пункт меню `Правка` -> `Уровень...`

Reviewer: ```suggestion
3. Нажмите `Ctrl` + `G` или выберите в меню `Правка` -> `Уровень...`
```

4. Поставьте галочку напротив пункта `Одновременно`.
5. Перетащите левый слайдер вниз.

>    - Уровень громкости в дБ (децибелах) будет изменяться с движением слайдера — подберите подходящую вам громкость.

Reviewer: сокращение имеет смысл оставить, если оно используется в интерфейсе, а так, кмк, оно только мешается

Author: В программе вообще сокращение на английском:

<img width="1125" height="548" alt="Image" src="https://github.com/user-attachments/assets/7073cf92-9f00-4808-8710-e869dbb919e3" />

Reviewer: тогда 
```suggestion
   - Уровень громкости в децибелах (dB) будет изменяться с движением слайдера — подберите подходящую вам громкость.
```

6. Когда вы будете довольны результатом, нажмите `OK`.

> 7. Нажмите `Ctrl` + `W` или выберите пункт меню `Файл` -> `Сохранить всё аудио...`

Reviewer: ```suggestion
7. Нажмите `Ctrl` + `W` или выберите в меню `Файл` -> `Сохранить всё аудио...`
```
